<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1252</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Восстановление фазы сигнала алгоритмом Фиенупа</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>231</width>
      <height>641</height>
     </rect>
    </property>
    <property name="title">
     <string>Параметры сигнала</string>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>191</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>Купол №1</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>A =</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_A1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>σ =</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_q1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>μ =</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_n1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>210</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>191</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>Купол №2</string>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>A =</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_A2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>σ =</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_q2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>μ =</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_n2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>500</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>390</y>
       <width>191</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>Купол №3</string>
     </property>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>A =</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_A3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>3.5</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>σ =</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_q3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>μ =</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_n3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>800</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>570</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>N =</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_N">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>570</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>1024</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>600</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>fd =</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_fd">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>600</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="signal_graph" native="true">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>941</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="spectrum_graph" native="true">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>320</y>
      <width>941</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="executeBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>700</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Сгенерировать сигнал</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>630</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Энергия&lt;br/&gt;разности&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_enery_delta">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>640</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1252</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
